src/main/java/net/glowstone/entity/GlowLightningStrike.java

```diff
@@ -113,6 +119,17 @@ public void pulse() {
                         ((Damageable) entity)
                             .damage(5, this, EntityDamageEvent.DamageCause.LIGHTNING);
                     }
+                    if (entity instanceof Pig) {
+                        GlowPig pig = (GlowPig) entity;
+                        GlowPigZombie zombie = EntityStorage.create(GlowPigZombie.class, location);
```

Comment:
Should other attributes like custom name be copied? Should an equipped saddle be dropped?

Reply:
Tested this in a SinglePlayer vanilla save w/ /entitydata commanf, only the customName is preserved. Potion effects, Inventory, Motion and Attributes are overwritten.